src/main/java/com/mycompany/app/App.java

```diff
@@ -2,16 +2,31 @@
 
 import javax.crypto.Cipher;
 import java.lang.Exception;
+import java.security.SecureRandom;
+
 
 /**
  * Hello world!
  *
  */
 public class App
 {
+    private static String AWS_SECRET_ACCESS_KEY = "gimme security issue!";
+    private static String username = "Gimme security issue!";
+    private static String password = "Gimme security issue!";
+    
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
+        System.out.format("No %s",1);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses &#39;%s&#39; to format int: . This is a potential locale-sensitive handling issue. It might cause errors in the handling and processing of the statement at line: 21. Consider formatting this data with &#39;%d&#39; instead.